<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Figkey Ip Capture Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/images/resource/icons/main_window.ico</normaloff>:/images/resource/icons/main_window.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDdd">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen_Test"/>
    <addaction name="actionSave_Test"/>
    <addaction name="actionSave_Server_Test"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionNetwork_Card"/>
    <addaction name="actionDoIP"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionFilter"/>
    <addaction name="actionFilter_Clear"/>
    <addaction name="actionVehicle_Identify"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionClient"/>
    <addaction name="actionServer"/>
    <addaction name="separator"/>
    <addaction name="actionSimulation_Client"/>
    <addaction name="actionSimulation_Server"/>
    <addaction name="actionSender"/>
   </widget>
   <addaction name="menuDdd"/>
   <addaction name="menuView"/>
   <addaction name="menuCapture"/>
   <addaction name="menuTool"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string>TCP assistants, including servers or clients</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen_Test"/>
   <addaction name="actionSave_Test"/>
   <addaction name="actionSave_Server_Test"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionFilter"/>
   <addaction name="actionFilter_Clear"/>
   <addaction name="actionVehicle_Identify"/>
   <addaction name="separator"/>
   <addaction name="actionClient"/>
   <addaction name="actionServer"/>
   <addaction name="actionSimulation_Client"/>
   <addaction name="actionSimulation_Server"/>
   <addaction name="actionSender"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/capture_stop.png</normaloff>:/images/resource/icons/capture_stop.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start Capture</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/capture_pause.png</normaloff>:/images/resource/icons/capture_pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause Capture</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/capture_start.png</normaloff>:/images/resource/icons/capture_start.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Capture</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/capture_filter.png</normaloff>:/images/resource/icons/capture_filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Capture Filter </string>
   </property>
  </action>
  <action name="actionFilter_Clear">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/capture_nofilter.png</normaloff>:/images/resource/icons/capture_nofilter.png</iconset>
   </property>
   <property name="text">
    <string>Filter Clear</string>
   </property>
   <property name="toolTip">
    <string>Capture Filter Clear</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/db_save.png</normaloff>:/images/resource/icons/db_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save to file</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/db_open.png</normaloff>:/images/resource/icons/db_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open from file</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/client.png</normaloff>:/images/resource/icons/client.png</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
   <property name="toolTip">
    <string>Support simulated udp/tcp client</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/server.png</normaloff>:/images/resource/icons/server.png</iconset>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
   <property name="toolTip">
    <string>Support simulated udp/tcp server</string>
   </property>
  </action>
  <action name="actionOpen_Test">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/open_test.png</normaloff>:/images/resource/icons/open_test.png</iconset>
   </property>
   <property name="text">
    <string>Open Test</string>
   </property>
   <property name="toolTip">
    <string>Load test data from json file and start testing</string>
   </property>
  </action>
  <action name="actionSave_Test">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/client_test.png</normaloff>:/images/resource/icons/client_test.png</iconset>
   </property>
   <property name="text">
    <string>Save Client Test</string>
   </property>
   <property name="toolTip">
    <string>Save client test data to json file</string>
   </property>
  </action>
  <action name="actionSave_Server_Test">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/server_test.png</normaloff>:/images/resource/icons/server_test.png</iconset>
   </property>
   <property name="text">
    <string>Save Server Test</string>
   </property>
   <property name="toolTip">
    <string>Save  server test data to json file</string>
   </property>
  </action>
  <action name="actionSimulation_Client">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/simulation_client.png</normaloff>:/images/resource/icons/simulation_client.png</iconset>
   </property>
   <property name="text">
    <string>Simulation Client</string>
   </property>
  </action>
  <action name="actionSimulation_Server">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/simulation_server.png</normaloff>:/images/resource/icons/simulation_server.png</iconset>
   </property>
   <property name="text">
    <string>Simulation Server</string>
   </property>
  </action>
  <action name="actionSender">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/sender.png</normaloff>:/images/resource/icons/sender.png</iconset>
   </property>
   <property name="text">
    <string>Sender</string>
   </property>
  </action>
  <action name="actionVehicle_Identify">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/vehicle_identify.png</normaloff>:/images/resource/icons/vehicle_identify.png</iconset>
   </property>
   <property name="text">
    <string>Vehicle Identify</string>
   </property>
   <property name="toolTip">
    <string>Automatically identify vehicle information</string>
   </property>
  </action>
  <action name="actionNetwork_Card">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/select_device.png</normaloff>:/images/resource/icons/select_device.png</iconset>
   </property>
   <property name="text">
    <string>Network Card</string>
   </property>
  </action>
  <action name="actionDoIP">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/resource/icons/doip_setting.png</normaloff>:/images/resource/icons/doip_setting.png</iconset>
   </property>
   <property name="text">
    <string>DoIP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
